plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -1,17 +1,46 @@
 import RestModel from "discourse/models/rest";
 import User from "discourse/models/user";
 import EmberObject from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { TrackedObject } from "@ember-compat/tracked-built-ins";
 
-export default class ChatMessage extends RestModel {}
+export default class ChatMessage extends RestModel {
+  @tracked id;
+  @tracked stagedId;
+  @tracked chatChannelId;
+  @tracked chatWebhookEvent;
+  @tracked cooked;
+  @tracked createdAt;
+  @tracked uploads;
+  @tracked excerpt;
+  @tracked message;
+  @tracked threadId;
+  @tracked reactions = new TrackedObject();
```

Comment:
Careful with this, since this is objects nested into objects I had to construct this myself to make it so all the tracked data worked correctly:

https://github.com/discourse/discourse/pull/20324/files#diff-97cdefa57def4cf645916acfa1c90a055cff3e567e0709879e8408bfc191ccc3R16-R23